keanu-project/src/main/java/io/improbable/keanu/network/VariableState.java

```diff
@@ -0,0 +1,4 @@
+package io.improbable.keanu.network;
+
+public interface VariableState {
```

Comment:
I'm not a big fan of marker interfaces, could we create a `VariableState` annotation and add it to the `VertexState` class?

Reply:
done